win32ss/user/ntuser/defwnd.c

```diff
@@ -529,6 +529,19 @@ DefWndScreenshot(PWND pWnd)
     UserCloseClipboard();
 }
 
+/* Is Window Snap enabled? */
+BOOL IsWindowSnapEnabled(VOID)
+{
+    WCHAR szValue[16];
```

Comment:
Why 16 elements? An arbitrary choice? We need only "0" / "1" plus maybe some UNICODE_NULL here, I guess.

Reply:
Fixed as `2`.